packages/backend/seeds/schedules-and-shifts.ts

```diff
@@ -39,50 +39,49 @@ export async function seed(knex: Knex): Promise<void> {
   ]);
 
   // Add 90 minute shifts for all hours.
-  await knex('shifts').insert(
-    generateShiftsAtIntervalOverRange(
-      90,
-      new Date('August 26, 2024 18:00'),
-      new Date('August 29, 2024 18:00'),
-      1,
-      1,
-    ),
+  const wenchShifts = generateShiftsAtIntervalOverRange(
+    90,
+    new Date('August 26, 2024 18:00'),
```

Comment:
As an alternative to hardcoding a list of shifts and schedule that isn't as difficult as an admin interface... you could have some JSON spec file that is read and the shifts are based on that

Reply:
Yeah thats sorta what i was thinking. For now we just needed some data in the DB and the seeds help a bunch.